### PR TITLE
Replace relay-pkg.so with relay.so in Dockerfile for Alpine

### PR DESCRIPTION
**Pull request by yatsukhnenko**
As discussed I will split #100 into smaller PRs.

### CODE REVIEW

File: docker/alpine.Dockerfile [hunk 1]
```diff
@@ -18,14 +18,31 @@ RUN apk add \
...
 ARG RELAY=v0.6.6
 
+RUN set -eux \
+  && apk add --no-cache --virtual .build-deps \
+  gcc \
+  make \
+  musl-dev \
+  openssl-dev \
+  # Build and install hiredis (>= 1.1.0) with SSL support \
+  && curl -L https://github.com/redis/hiredis/archive/refs/tags/v1.2.0.tar.gz | tar -xz \
```
Comment (szepeviktor) [inline docker/alpine.Dockerfile:28]: Nice to see that you put the _glue_ at the front of the line.
Now it causes no brain damage 🧠

**Comment by szepeviktor** [comment]
@yatsukhnenko Do inline comments work in Dockerfiles?

File: docker/alpine.Dockerfile [hunk 2]
```diff
@@ -18,14 +18,31 @@ RUN apk add \
...
+  && PREFIX=/usr USE_SSL=1 make -C hiredis-1.2.0 install \
+  # Build and install concurrencykit \
+  && curl -L https://github.com/concurrencykit/ck/archive/refs/tags/0.7.1.tar.gz | tar -xz \
```
Comment (tillkruss) [inline docker/alpine.Dockerfile:31]: Let's install `ck 0.7.1` via Alpine 3.18.

File: docker/alpine.Dockerfile [hunk 3]
```diff
@@ -18,14 +18,31 @@ RUN apk add \
...
+  && cd ck-0.7.1 \
+  && ./configure \
+  && make install \
+  # Cleanup \
+  && apk del .build-deps
+
 # Download Relay
 RUN PLATFORM=$(uname -m | sed 's/_/-/') \
   && curl -L "https://builds.r2.relay.so/$RELAY/relay-$RELAY-php8.0-alpine3.9-$PLATFORM.tar.gz" | tar xz -C /tmp
```
Comment (tillkruss) [inline docker/alpine.Dockerfile:40]: We need to use:

```
https://builds.r2.relay.so/$RELAY/relay-$RELAY-php8.1-alpine3.17-$PLATFORM.tar.gz
```

**Comment by yatsukhnenko** [comment]
@szepeviktor yes, because comment lines are removed before the Dockerfile instructions are executed

**Pull request merged**